.github/workflows/cmake.yml

```diff
@@ -21,6 +21,15 @@ jobs:
     - uses: actions/checkout@v3
       with:
          submodules: 'recursive'
+         fetch-depth: '0'
+    - run: |
+        git fetch --prune --tags
+        git tag --list
+        git fetch --all --tags
+        SD2PSX_VERSION=$(git describe --exclude latest | sed 's/\(.*\)-.*/\1/')
```

Comment:
I think this should go in a separate pr. seems it would require annotated tags? and what happens for automatic builds for commits which aren't tagged? tbh once we start tagging releases I'd rather post them myself, rather than it being automatic

Reply:
Fixed - now it either uses the Tag as the version or created a build called ```nightly-<<COMMIT_HASH>>```